modules/nw-metallb-installing-operator-cli.adoc

```diff
@@ -101,7 +99,7 @@ metadata:
   name: metallb-operator-sub
   namespace: metallb-system
 spec:
-  channel: "${OC_VERSION}"
+  channel: "stable"
```

Comment:
is it always stable? Shouldn't be 4.10 something?

Reply:
I checked and this command `oc get packagemanifest metallb-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel} ` returned `stable`. I know around 4.10 timeframe some Operators were moved to stable channel. Maybe @evgenLevin can verify, but I recall making updates to other Operators from `OC_VERSION` to stable in 4.10 release

Reply:
I tried 4.11 and MetalLB Operator failed to install. It only seems to install with channel set to `stable`.

Reply:
That's because 4.11 is not out yet

Reply:
OK going to try with 4.10 I maybe mistakenly thought channel was updated to stable for this Operator from 4.10 >

Reply:
reverted to original content as you state works with 4.10